table/two_level_iterator.cc

```diff
@@ -158,14 +158,9 @@ void TwoLevelIterator::Prev() {
 
 
 void TwoLevelIterator::SkipEmptyDataBlocksForward() {
-  while (second_level_iter_.iter() == nullptr ||
```

Comment:
My original intention was to make things clearer. But it looks like there are not much assumptions that can be made. I find the resulting code hard to follow and understand. For me it's not clear if all conditions within the loop can happen independently of the parts of the while condition of if they are somehow connected. Though it sounds like they are independent, so things need to stay like this.